### PR TITLE
Sign up page

### PR DESCRIPTION
**Pull request by jdavidrice**
## Proposed changes
The basic framework for a sign-up page. May need re-styling after discussion with team. As with the SignIn page, this still needs to be tied into the backend. It is being delivered at this time to allow for group discussion. 

## PR Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/davidsaulrodriguez/shop-portal/blob/master/CONTRIBUTING.md) doc
- [x] I have read the [BRANCHING MODEL](https://github.com/davidsaulrodriguez/shop-portal/blob/master/BRANCHINGMODEL.md) doc
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## PR Type

What types of changes does your code introduce to **Shop Portal**?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code Style Update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (Please describe):

## Current behavior

Issue Number: #31

## New behavior

Fixes N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -13,6 +14,9 @@ function App() {
       <Switch>
         <Route exact from='/login' render={(props) => <SignIn {...props} />} />
       </Switch>
+      <Switch>
```
Comment (davidsaulrodriguez) [inline client/src/App.jsx]: You can remove this switch statement. It isn't needed.

File: client/src/App.jsx [hunk 2]
```diff
@@ -13,6 +14,9 @@ function App() {
...
+        <Route exact from='/signup' render={(props) => <SignUp {...props} />} />
```
Comment (davidsaulrodriguez) [inline client/src/App.jsx:16]: You should move this line into the switch statement above it instead.

**Review by davidsaulrodriguez — CHANGES_REQUESTED** [review]
Once you remove the unneeded switch statement and move the _signup_ route into the preexisting switch statement this PR will be ready to merge.

**Comment by jdavidrice** [comment]
I have made the changes that you mentioned. Thank you!

**Review by davidsaulrodriguez — APPROVED** [review]
Everything looks great. Changes approved!

**Review by pennyquirino — APPROVED** [review]

**Pull request merged**